### ISSUE TITLE
Crash while using an editor

### ISSUE THREAD

**Issue by Yggddrassill** [4ian/GDevelop#4192]
=> Please write here a short description of when the error occured and how to reproduce it.
You also may have to create an account on GitHub before posting.

When you're ready, click on "Submit new issue". Don't change the rest of the message. Thanks!

## Error stack (don't write anything here)
```
RuntimeError: memory access out of bounds
    at file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/libGD.wasm?cache-buster=5.0.139-0ffc88a4706e4b5f2c96f6d1492d2f59eb2b2a6e:wasm-function[2862]:0x8fcd9
    at file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/libGD.wasm?cache-buster=5.0.139-0ffc88a4706e4b5f2c96f6d1492d2f59eb2b2a6e:wasm-function[437]:0x27624
    at Project.HasLayoutNamed.Project.HasLayoutNamed [as hasLayoutNamed] (file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/libGD.js?cache-buster=5.0.139-0ffc88a4706e4b5f2c96f6d1492d2f59eb2b2a6e:9:950593)
    at n.value (file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/static/js/1.8ef70361.chunk.js:1:435122)
    at n.value (file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/static/js/1.8ef70361.chunk.js:1:435256)
    at Ha (file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/static/js/64.e29dea9d.chunk.js:2:406147)
    at Ba (file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/static/js/64.e29dea9d.chunk.js:2:405940)
    at ks (file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/static/js/64.e29dea9d.chunk.js:2:441563)
    at mu (file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/static/js/64.e29dea9d.chunk.js:2:433005)
    at vu (file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/static/js/64.e29dea9d.chunk.js:2:432930)
    at su (file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/static/js/64.e29dea9d.chunk.js:2:429939)
    at file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/static/js/64.e29dea9d.chunk.js:2:381417
    at t.unstable_runWithPriority (file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/static/js/64.e29dea9d.chunk.js:2:456644)
    at $i (file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/static/js/64.e29dea9d.chunk.js:2:381126)
    at Qi (file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/static/js/64.e29dea9d.chunk.js:2:381362)
    at Ji (file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/static/js/64.e29dea9d.chunk.js:2:381297)
    at nu (file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/static/js/64.e29dea9d.chunk.js:2:426721)
    at ba (file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/static/js/64.e29dea9d.chunk.js:2:398512)
    at file:///C:/Users/ahhue/AppData/Local/Programs/GDevelop/resources/app.asar/www/static/js/1.8ef70361.chunk.js:1:369529
```

## Component stack (don't write anything here)
```

    in n
    in t
    in Unknown
    in Unknown
    in div
    in n
    in Unknown
    in div
    in Unknown
    in n
    in d
    in j
    in f
    in j
    in h
    in g
    in c
    in n
    in t
    in be
    in n
    in Unknown
    in g
    in t
    in l
    in o
    in n
    in n
    in DragDropContext(n)
    in n
    in c
```

## Other details
* IDE version: 5.0.139-0ffc88a4706e4b5f2c96f6d1492d2f59eb2b2a6e
* Arch: x64,
* Platform Name: win32,
* System Version: 10.0.19043,
* User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) GDevelop5/5.0.139 Chrome/100.0.4896.143 Electron/18.2.2 Safari/537.36,

**Comment by Bouh** [comment]
Hi @Yggddrassill 
Can you add a brief description of when the error occurred?
If you have an idea of how to reproduce the problem, or a project, that might help.
Thanks

**Comment by Yggddrassill** [comment]
It occurs while trying to run the prevew most of the times after adding events, sometimes it happens when coppied events have lots of variables being checked to trigger one event, the later was workedaround by lowering the amount of variables. Most of the times It is impredictable but the result is either a black screen, a box to report the issue at github, or a text saying that there Is no lay out for the scene and the only thing that can be clicked on is the close button.
My game  has lots of objects that use variables as stats or attributes, if an objet is by example with a fire object, the program Will check if the object is of a kind that Is either killed, powered up, or hurt by it. There aré many attributes resistant to fire, and when to many events called for all of those attributes  to be checked by the same trigger, the error occured. The work around was making a FireResistant attribute and adding it on all of the fire resisting objects. It wasnt giving any trouble until I duplicated the events on a second trigger to add the "trigger only once" condition. It was very weird foto the prevew to crash at start up, because the scene containing those events is oppened by plugging more than
